task/github.py

```diff
@@ -121,17 +122,24 @@ def __init__(self, base=None, cacher=None, repo=None):
         self.token = None
         self.debug = False
         try:
-            with open(xdg_config_home('github-token'), "r") as f:
+            with open(xdg_config_home('cockpit-dev', 'github-token'), "r") as f:
                 self.token = f.read().strip()
         except FileNotFoundError:
-            # fall back to GitHub's CLI token
             try:
-                with open(xdg_config_home("gh/config.yml")) as f:
-                    match = re.search(r'oauth_token:\s*(\S+)', f.read())
-                if match:
-                    self.token = match.group(1)
+                with open(xdg_config_home('github-token'), "r") as f:
+                    warnings.warn('~/.config/github-token is deprecated.  '
+                                  'Please move to ~/.config/cockpit-dev/github-token.',
```

Comment:
Please drop the final period, so that this can be copy&pasted, and there is no ambiguity what is part of the file name and what is merely part of the sentence.

Reply:
Sure.  But let's chat tomorrow about if we just drop this commit or not.